packages/argo-run/src/dev.ts

```diff
@@ -239,18 +244,24 @@ export async function dev(...args: string[]) {
 
   await Promise.all([firstCompilePromise, httpListenPromise]);
 
-  log(`Your extension is available at ${localhostScriptUrl}`);
+  log(`Your extension is available at ${scriptUrl}`);
```

Comment:
Changed the message to `Starting dev server on port ${port} …` because this step could theoretically fail and it would be good to know that it likely had to do with starting the server. This results in the following output:

```
yarn run v1.22.10
$ yarn start --publicUrl http://1234.ngrok.io
$ argo-run serve --port 39351 --publicUrl http://1234.ngrok.io
🔭 > Starting dev server on port 39351 …
🔭 > Your extension is available at http://1234.ngrok.io/assets/extension.js
🔭 > You can append this query string: script_url=http%3A%2F%2F1234.ngrok.io%2Fassets%2Fextension.js&api_key=6f44585b338911f3fa53408e41871db2
```